test/helpers/url-helpers.test.ts

```diff
@@ -0,0 +1,101 @@
+import * as url_helpers from "../../src/helpers/url-helpers"
+
+describe("url_helpers.isValidExternalURL", () => {
+    test("0", () => {
+        let result: any = url_helpers.isValidExternalURL({ toString: () => "2019-07-01" })
+        expect(result).toMatchSnapshot()
```

Comment:
I think calling `toMatchSnapshot()` is incorrect. We should assert the expected and actual values imo